banking_system/user_management/custom_backend.py

```diff
@@ -3,10 +3,11 @@
 
 
 class CustomAuthBackend(ModelBackend):
-    def authenticate(self, request, username=None, password=None):
+    def authenticate(self, request, email=None, password=None):
```

Comment:
Did you test this ? When I did this, the request never came here.

Reply:
request was already there. I just changed the parameter name from username to email. I have tested it, working fine.